Editor/Scripts/interactivetutorials_dialog.py

```diff
@@ -191,7 +197,7 @@ def update_step_view(self):
 
         self.title_label.setText(self.current_step.get_title())
         self.content_area.setText(self.current_step.get_content())
-
+        self.step_label.setText(f"Step {self.current_step_index} of {self.current_tutorial_num_steps}")
```

Comment:
Set step 0 to "Click next to continue." per previous suggestion.

```suggestion
        self.step_label.setText("Click next to continue.")
        if self.current_step_index > 0:
            self.step_label.setText(f"Step {self.current_step_index} of {self.current_tutorial_num_steps}")